Dockerfile.slim

```diff
@@ -17,13 +17,15 @@ RUN apk update \
     bash \
     ca-certificates \
     wget \
-    openjdk11
+    openjdk11 \
+    git \
```

Comment:
Seems like some things are pinned, and some not.  I'd be tempted to pin at least git in the spirit of reproducible builds.

Reply:
+1. I think `SSH_ASKPASS` and `GIT_SSH_COMMAND` are only available after certain versions. Could be wrong, haven't looked it up in a while.

Reply:
According to [Alpine package index](https://pkgs.alpinelinux.org/packages?name=git&branch=v3.12), the current git version included in Alpine 3.12 is `2.26.2-r0`, so this is the "minimum" version to work with. I tested that `SSH_ASKPASS` and `GIT_SSH_COMMAND` work in this version.

We're not pinning any other Alpine package in clouddriver (but we pin externally installed resources like kubectl and aws-cli). Two disadvantages of pinning git: 

* At some point in the future it's possible that the build breaks, by not finding an old git version in Alpine repositories for the given Alpine version. Alpine doesn't store all package versions forever.
* We would not get CVE fixes automatically, and instead will have to manually update the version.

Advantages of pinning:

* We protect ourselves if in future versions git drops support of something used by clouddriver, which could be a really hard to troubleshoot bug.

I think pinning makes more sense, I would hate if git changes the flags for shallow clones and we suddenly see clouddriver failing when there were no code changes at all (been there).